pkg/controller/manage/manage_controller.go

```diff
@@ -225,6 +228,7 @@ func enableKmeshManage(pod *corev1.Pod, queue workqueue.RateLimitingInterface, s
 		return
 	}
 	queue.AddRateLimited(QueueItem{podName: pod.Name, podNs: pod.Namespace, action: ActionAddAnnotation})
+	linkXdp(nspath, xdpProgFd, mode)
```

Comment:
should add a unlink op in `disableKmeshManage`

Reply:
done